string[] ids    #### Vector of SOMA2 ids
---
bool result
